src/main/app/controller/AppRequest.ts

```diff
@@ -15,6 +14,6 @@ export interface AppRequest<T = Record<string, unknown>> extends Request {
 
 export interface AppSession extends Session {
   lang: string,
-  errors: DefinedError[] | undefined,
+  errors: [] | undefined,
```

Comment:
what does `[]` even mean? array of what?

Reply:
ill make a type, plan was to have { propertyName, errorType }